.circleci/config.yml

```diff
@@ -1,28 +1,52 @@
 version: 2
+
 jobs:
   unit:
     docker:
-      - image: fishtownjacob/test-container
+      - image: fishtownjacob/test-container:4
+        environment:
+          DBT_INVOCATION_ENV: circle
     steps:
       - checkout
       - run: tox -e flake8,unit
```

Comment:
This one is failing because the fishtownjacob/test-container does not contain `libsasl2-dev`.
Switching to dmateusp/test-container will probably fix it